coalahtml/_coalahtml/app/controllers/logs.js

```diff
@@ -2,12 +2,15 @@
 
 angular.module('coalaHtmlApp')
   .controller('LogsCtrl', function ($scope, $http) {
+    $http.get("data/Constants.json").success(function(content) {
+      $scope.Constants = content;
+      parseCoalaProject($scope.Constants.data);
+    });
```

Comment:
@Uran198 Can't. Because each controller would be processed separately depending on the route request. Right?  :smile: